core/commands/repo.go

```diff
@@ -290,9 +287,12 @@ var repoVerifyCmd = &cmds.Command{
 				out <- &VerifyProgress{Progress: i}
 			}
 			if fails == 0 {
+				log.Debug("Run.go.else: all ok")
 				out <- &VerifyProgress{Message: "verify complete, all blocks validated."}
 			} else {
-				out <- &VerifyProgress{Message: "verify complete, some blocks were corrupt."}
+				//out <- &VerifyProgress{Message: "verify complete, some blocks were corrupt."}
+				log.Debugf("Run.go.else: %T.SetError", res)
+				res.SetError(fmt.Errorf("verify complete, some blocks were corrupt"), cmdsutil.ErrNormal)
```

Comment:
Two thoughts. First, this is the old `Response` and I didn't change it too much. Here we still have to pass `error, ErrorType`. With the new ones it is `interface{}, ErrorType`. Second, making it accept only a string (or `interface{}` for that matter) seems kind of sketchy because the only way to do it is to make it variadic and thus accept more than one `ErrorType` which is also weird:
```go
func (re *ResponsEmitter) SetError(msg interface{}, codes ...ErrorType)
```